models/articles.model.js

```diff
@@ -48,4 +48,17 @@ exports.updateVotes = (article, voteIncrement) => {
         else
         return({"newArticle" : rows})
        })
+}
+exports.addUserComment = (username,body,article_id) => {
+      return db.query(`
+      INSERT INTO comments
+      (body,author,article_id)
+      VALUES
+      ($1, $2, $3) RETURNING *;`, [body,username,article_id])
+      .then(({rows}) => {
+        if(!body || !username) {
```

Comment:
We don't need to worry about a custom error here, sql will throw an error if it needs to as the post will violate the foreign key restraint referencing the users table. Hint: This will be the same for when you write your id tests. It's always good to try and pass your error to sql first to see if it's going to throw it's own error before writing a custom error.